js/binance.js

```diff
@@ -4930,4 +4930,57 @@ module.exports = class binance extends Exchange {
             'info': response,
         };
     }
+
+    async fetchBorrowInterestHistory (code = undefined, symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        this.loadMarkets ();
+        const request = {};
+        let market = undefined;
+        if (code !== undefined) {
+            request['asset'] = this.currency (code)['id'];
+        }
```

Comment:
We're going to unify and detach the parser eventually, and the parser will need a currency structure to work.
For that reason, let's always write the above three lines 4938-4940 in the following manner:

```JavaScript
let currency = undefined;
if (code !== undefined) {
    currency = this.currency (code);
    request['asset'] = currency['id'];
}
```